core-web/libs/data-access/src/lib/dot-properties/dot-properties.service.ts

```diff
@@ -57,13 +57,14 @@ export class DotPropertiesService {
 
     /**
      * Get the value of specific feature flag
-     *
      * @param {FeaturedFlags} key
      * @return {*}  {Observable<boolean>}
      * @memberof DotPropertiesService
      */
     getFeatureFlag(key: FeaturedFlags): Observable<boolean> {
-        return this.getKey(key).pipe(map((value) => value === 'true'));
+        return this.getKey(key).pipe(
+            map((value) => (value === FEATURE_FLAG_NOT_FOUND ? true : value === 'true'))
```

Comment:
So this means that if we hit a FF that is not in the env we anyways will show it? 
What if a client doesn't config those FF in its env? 
Can we add test cases for that?

I am not sure if we really want to show the feature if the flag is not found all the time.

Reply:
@zJaaal There is a unit test for this service but I case of `FEATURE_FLAG_NOT_FOUND` It needs to behave exactly as true based on the meeting had with @fmontes yesterday.